lib/android/src/main/java/com/airbnb/android/react/navigation/DefaultNavigationImplementation.java

```diff
@@ -260,14 +260,14 @@ public void reconcileNavigationProperties(
       foregroundColor = next.getInt("foregroundColor");
     }
 
-    if (stringHasChanged("title", prev, next)) {
+//    if (stringHasChanged("title", prev, next)) {
```

Comment:
Seems more like `stringHasChanged` should be updated so that if one value is `null` it should return `true`?

(I mean, this is fine too, but doesn't address the underlying issue that probably impacts other parameters?)